### PR TITLE
Host emu

### PR DESCRIPTION
**Pull request by ravi-rahul**

### CODE REVIEW

File: cmake/common.cmake [hunk 1]
```diff
@@ -146,95 +134,27 @@ include_directories(${PROJECT_SOURCE_DIR}
...
+set(SYSTEM_LINK_LIBS
+    tivision_apps
+    edgeai-tiovx-kernels
+    edgeai-apps-utils
+    m
+    yaml-cpp
+    )
 
 if ("${TARGET_OS}" STREQUAL "LINUX")
+if ("${TARGET_CPU}" STREQUAL "A72" AND "${TARGET_CPU}" STREQUAL "A53")
```
Comment (sinha-shreyash) [inline cmake/common.cmake:146]: @ravi-rahul , It should be ${TARGET_CPU}" STREQUAL "A72" **OR** "${TARGET_CPU}" STREQUAL "A53"

File: modules/CMakeLists.txt [hunk 2]
```diff
@@ -28,7 +28,7 @@ set(SRC_FILES
     src/tiovx_lut_module.c
     )
 
-if ("${TARGET_OS}" STREQUAL "LINUX")
+if ("${TARGET_OS}" STREQUAL "LINUX" AND "${TARGET_CPU}" STREQUAL "A72" AND "${TARGET_CPU}" STREQUAL "A53")
```
Comment (sinha-shreyash) [inline modules/CMakeLists.txt:31]: here too same as above

File: tests/CMakeLists.txt [hunk 3]
```diff
@@ -23,7 +23,7 @@ set(SRC_FILES
     app_tiovx_fakesrc_fakesink_module_test.c
     main.c)
 
-if ("${TARGET_OS}" STREQUAL "LINUX")
+if ("${TARGET_OS}" STREQUAL "LINUX" AND "${TARGET_CPU}" STREQUAL "A72" AND "${TARGET_CPU}" STREQUAL "A53")
```
Comment (sinha-shreyash) [inline tests/CMakeLists.txt:26]: same as above

File: tests/CMakeLists.txt [hunk 4]
```diff
@@ -44,7 +44,7 @@ if ("${TARGET_SOC}" STREQUAL "J721E" OR "${TARGET_SOC}" STREQUAL "J721S2" OR "${
          app_tiovx_dof_module_test.c
          app_tiovx_capture_dl_display_pipeline.c)
 
-         if ("${TARGET_OS}" STREQUAL "LINUX")
+         if ("${TARGET_OS}" STREQUAL "LINUX" AND "${TARGET_CPU}" STREQUAL "A72" AND "${TARGET_CPU}" STREQUAL "A53")
```
Comment (sinha-shreyash) [inline tests/CMakeLists.txt:47]: same as above

File: cmake/common.cmake (hunk 1 above)
Comment (ravi-rahul) [reply cmake/common.cmake]: Thanks for finding this @sinha-shreyash 
Fixed this now

File: tests/CMakeLists.txt (hunk 3 above)
Comment (ravi-rahul) [reply tests/CMakeLists.txt]: Fixed

File: tests/CMakeLists.txt (hunk 4 above)
Comment (ravi-rahul) [reply tests/CMakeLists.txt]: Fixed

**Review by ankur-ti — APPROVED** [review]

**Pull request merged**